Source/diablo.cpp

```diff
@@ -1099,6 +1101,8 @@ void PressChar(int vkey)
 		return;
 	}
 	if (PauseMode == 2) {
+		if (GetAsyncKeyState(VK_MENU) & 0x8000)

```

Comment:
This appears to be the wrong place for this, it's not fitering for enter, it looks like alt+anything (except P) will toggle fullscreen with this code. Possibly the event listener used when paused is where this belongs.

Reply:
you are right, just tested :P

Reply:
Now it work perfect

Reply:
still bad indentation ... >.>

Reply:
How I can improve the indentation?

Reply:
Look at any other if in the code ? >.>

Reply:
Which part is misaligned.
The top part or the bottom part or both?
All columns appear to be on the same level.

Reply:
that's the reason why it's wrong

Reply:
Which line number?

Reply:
... Too far left?
... Too far to the right?

Reply:
you need a tab before `dx_reinit();` ;(

Reply:
done

Reply:
yay

Reply:
Is it now acceptable master qndel?